src/ui/history_view.rs

```diff
@@ -70,19 +83,32 @@ fn history_line(
 				EventPropagation::Continue
 			}),
 		dyn_container(
-			move || view_button_switch.get(),
-			move |switch| {
-				if switch {
-					container(
-						scroll(label(move || field_value.get()))
-							.style(|s| s.flex_grow(1.0).width(80)),
+			move || (view_button_switch.get(), field_value.get()),
```

Comment:
This was tricky to figure out. Needed two signals here to cause this component to re-render

Reply:
Oh thats cool you can tuple them.